eclipse-scout-core/src/jquery/jquery-scout-types.ts

```diff
@@ -617,14 +617,16 @@ declare global {
     isAttached(): boolean;
 
     /**
+     * @param includePseudoScrollParents whether pseudo scrollable elements should be considered or not
```

Comment:
Maybe add a short explanation what "pseudo scrollable" elements are so users of this method don't have to analyze the code.